### ISSUE TITLE
[1.38.0-beta.1]  Closing modal after dismissing an issue is difficult

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1474]
After dismissing an issue, focus is shifted to the success message. There are then multiple tab stops to be able to close the modal with the keyboard:

<img width="923" height="785" alt="Image" src="https://github.com/user-attachments/assets/92ecfc1b-abd7-4eed-adf4-1ea9ecc6e10c" />
1. Dismiss notice button
2. Reopen issue button
3. Document `.components-modal__content is-scrollable.has-scrolled-content`
4. Close modal button

This seems like it could be improved. Ideas:
- Put a link-styled button in the success notice "Close modal"
- Add the option to dismiss and close the modal in free (currently, this is only a pro feature)
